sw/device/lib/dif/dif_kmac_unittest.cc

```diff
@@ -86,5 +90,90 @@ TEST(FunctionNameTest, BadArg) {
   EXPECT_EQ(dif_kmac_function_name_init("", -1, &fn), kDifBadArg);
 }
 
+using mock_mmio::MmioTest;
+using mock_mmio::MockDevice;
+using testing::Test;
+
+// Base class for the rest fixtures in this file.
+class KmacTest : public testing::Test, public mock_mmio::MmioTest {
+ public:
+  dif_kmac_t kmac_;
+  dif_kmac_operation_state_t op_state_;
+  const uint8_t kMsg_[17] = {0xa7, 0x48, 0x47, 0x93, 0x0a, 0x03,
```

Comment:
Since you aren't using this directly in any C code it might be better to make this a `std::array` and get rid of `kLen_`.